Automattic/sensei
# Add eslint jsdoc alignment plugin

### Changes proposed in this Pull Request

* It introduces a new [eslint plugin to validate the JSDoc alignment](https://github.com/Automattic/eslint-plugin-jsdoc-alignment), following the [WP standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#aligning-comments).
* This PR also already ran the plugin auto fix to fix the current unformatted JSDocs.

### Testing instructions

* Make sure the auto fixed JSDocs are correct.
* Edit/add a JSDoc, making the alignment wrong, like that:
```
/**
 * Function description.
 *
 * @param {string} lorem Description.
 * @param {int}    sit        Description multi words.
 */
```
* Run `npm run lint-js` and make sure the error is logged in the terminal.
* Run `npm run lint-js:fix` and make sure it was automatically fixed.
